assets/js/base/hooks/use-store-products.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import { COLLECTIONS_STORE_KEY as storeKey } from '@woocommerce/block-data';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { useShallowEqual } from './use-shallow-equal';
+
+const DEFAULT_OPTIONS = {
+	namespace: '/wc/blocks',
+	modelName: 'products',
```

Comment:
Nitpicking but is there a reason why we want for `model` rather than `route` for consistency with the API code?

Reply:
I mostly did it because it won't always be "route" (for example `products/attributes/terms`).  I feel `modelName` is a better reflection of the model entities the route returns.  Instead of aligning `modelName` to the route part (which is a true alignment only for routes without ids in the path), we could align with the `title` property on the route schema as that is usually the "official" modelName.  But the problem there, is the schema `title` is not available on the `GET {namespace}` routes response.

Reply:
Ok, so resource names then?

Reply:
Ya that could work better (`resourceName`).  Worth doing a pull to rename this everywhere (including data stores)?  I'm okay with that.